bmc/bmc.go

```diff
@@ -124,6 +142,17 @@ type Storage struct {
 	State common.State `json:"state,omitempty"`
 }
 
+type Storage struct {
+	Entity
+	State common.State `json:"state,omitempty"`
+	// Description provides a description of this resource.
+	Description string `json:"description,omitempty"`
```

Comment:
Do we really wan to have a `description` field in the status? Where is this information coming from and what should the controller do with this information? Or is it just for the information of the user?

Reply:
just in case vendors put some useful information in there ;)

can be removed

Reply:
Lets keep this out for now. We can always add fields. Removing is always a pain.

Reply:
it is anyway not in the server crd api, just our internal bmc storage structure.